From ddae789458cec5c5f2c57d433cb6628d72fe09cb Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Sat, 3 Dec 2016 14:34:55 +0000
Subject: [PATCH 149/268] Update European services in tf582_4.txt

Trailing ";" left on the end of NPL link by asciidoc
---
 docs/tf582_4.txt | 21 ++++++++-------------
 1 file changed, 8 insertions(+), 13 deletions(-)

diff --git a/docs/tf582_4.txt b/docs/tf582_4.txt
index 6a141b3..6cdb1ba 100644
--- a/docs/tf582_4.txt
+++ b/docs/tf582_4.txt
@@ -17,27 +17,22 @@ examples are:
 service. Phone number: +49 5 31 51 20 38.
 +
 For more detail, see
-http://www.ptb.de/english/org/4/43/433/disse.html[http://www.ptb.de/english/org/4/43/433/disse.htm]
-* In the UK by National Physical Laboratory (NPL)'s TRUETIME service.
-Phone number: 0891 516 333
-+
-For more detail, see http://www.npl.co.uk/npl/ctm/truetime.html
+http://www.ptb.de/cms/index.php?id=1786&L=1[http://www.ptb.de/cms/index.php?id=1786&L=1]
 
-* In Italy by Istituto Elettrotecnico Nazionale "Galileo Ferrais"
-(IEN)'s CTD service. Phone number: 166 11 46 15
+* In the UK by National Physical Laboratory (NPL)'s Telephone Time Service.
+Phone number: 020 8943 6333
 +
-For more detail, see http://www.ien.it/tf/time/Pagina42.html
+For more detail, see http://www.npl.co.uk/science-technology/time-frequency/products-and-services/time/time-synchronisation-of-computers-to-utc(npl&#41;
 
-* In Switzerland by Swiss Federal Office of Metrology's timecode
-service. Phone number: 031 323 32 25
+* In Italy by L'Istituto Nazionale di Ricerca Metrologica
+(INRIM)'s CTD service. Phone numbers: 011 39 19 263 and 011 39 19 264
 +
-For more detail, see
-http://www.metas.ch/en/labors/official-time/modem/index.html[http://www.ofmet.admin.ch/de/labors/4/Zeitvert.html]
+For more detail, see http://www.inrim.it/res/tf/ctd_i.shtml
 
 * In Sweden by SP Swedish National Testing and Research Institute 's
 timecode service. Phone number: +46 33 415783.
 +
-For more detail, see http://www.sp.se/metrology/timefreq/eng/tandf.htm
+For more detail, see http://www.sp.se/en/index/services/time_sync/modem_time/Sidor/default.aspx
 
 == Additional Information ==
 
-- 
2.7.4

